junit5-support/src/main/kotlin/in/specmatic/test/SpecmaticJUnitSupport.kt

```diff
@@ -110,7 +110,7 @@ class TestReport(private val testReportRecords: MutableList<TestResultRecord> =
         logger.newLine()
 
         val recordsWithFixedURLs = testReportRecords.map {
-            it.copy(path = it.path.replace(Regex("""\((.*):.*\)"""), "{$1}"))
+            it.copy(path = it.path.replace(Regex("""\((.*?):.*?\)"""), "{$1}"))
```

Comment:
Can you extract this functionality to a method and add a unit test.

The functionality of doing the regex replace across all 3 seems the same.. if so, can you extract the regex replace itself into a common method.